src/current/v25.1/admission-control.md

```diff
@@ -42,25 +42,43 @@ Admission control can help if your cluster has degraded performance due to the f
 
 Almost all database operations that use CPU or perform storage IO are controlled by the admission control system. From a user's perspective, specific operations that are affected by admission control include:
 
-- [General SQL queries]({% link {{ page.version.version }}/selection-queries.md %}) have their CPU usage subject to admission control, as well as storage IO for writes to [leaseholder replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases).
+- [General SQL queries]({% link {{ page.version.version }}/selection-queries.md %}) have their CPU usage subject to admission control, as well as storage IO for writes to [leaseholder replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) and [follower replicas](#replication-admission-control).
 - [Bulk data imports]({% link {{ page.version.version }}/import-into.md %}).
 - [`COPY`]({% link {{ page.version.version }}/copy.md %}) statements.
 - [Deletes]({% link {{ page.version.version }}/delete-data.md %}) (including deletes initiated by [row-level TTL jobs]({% link {{ page.version.version }}/row-level-ttl.md %}); the [selection queries]({% link {{ page.version.version }}/selection-queries.md %}) performed by TTL jobs are also subject to CPU admission control).
 - [Backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}).
 - [Schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), including index and column backfills (on both the [leaseholder replica]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) and [follower replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft)).
-- [Follower replication work]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft).
+- [Follower replication work](#replication-admission-control).
 - [Raft log entries being written to disk]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft).
```

Comment:
@kvoli is this 'raft log entries' sort of subsumed by 'follower replication work'? or does it make sense to continue to leave it alone as a separate item?

Reply:
`follower replication work` includes `Raft log entries being written to disk` but the leader also paces writing log entries to disk.

Having said that, I'd lean towards leaving both despite the overlap because it informs the reader to think about both.

Reply:
👍 - thanks, will leave both in that case